### PR TITLE
Add a new rebus.

### PR DESCRIPTION
**Pull request by zangiboy**
Associated Issue: #<34>

### Summary of Changes

- Add a new rebus for Antarctica
-

### CODE REVIEW

File: src/js/rebuses.js [hunk 1]
```diff
@@ -175,6 +175,10 @@ const rebuses = [
     symbols: ['sc', '+', '🦍', '+', '🐐'],
     words: ['scapegoat']
   }
```
Comment (ollelauribostrom) [inline src/js/rebuses.js]: Looks like you forgot to add a `,` before your rebus object 🙂

File: src/js/rebuses.js [hunk 2]
```diff
@@ -174,6 +174,10 @@ const rebuses = [
   {
     symbols: ['sc', '+', '🦍', '+', '🐐'],
     words: ['scapegoat']
+  },
+  {
+    symbols: ['🐜', '+', '⛵️', '+' 'tica'],
```
Comment (ollelauribostrom) [inline src/js/rebuses.js]: Also: add a `,` after the `'+'` in your array

Comment (zangiboy) [reply src/js/rebuses.js]: Thanks for your patience with my careless typing!

**Comment by ollelauribostrom** [comment]
Awesome, thank you!

**Pull request merged**